### PR TITLE
Bring in missing PDD code

### PR DESCRIPTION
**Pull request by nickgraz**
It seems when I first brought the PDD code into `terraware-server`, I failed to update my local with `main` beforehand. 

My local was at `55c4bd1`, which is the merge commit for this PR - https://github.com/terraware/pdd-backend/pull/111

Other than update/upgrade PRs, there were 5 that were missed in the initial push of this code into `terraware-server`. Here are the `pdd-backend` PRs that contained net-new code and their relevant commit hash in this PR:
https://github.com/terraware/pdd-backend/pull/127 - [2ac3903](https://github.com/terraware/terraware-server/pull/2135/commits/2ac390348d2daced04ba1eed1220de66d4fc44cf)
https://github.com/terraware/pdd-backend/pull/128 - [2e8609c](https://github.com/terraware/terraware-server/pull/2135/commits/2e8609c67fd6b4434400c6aeaafa11766ece3161)
https://github.com/terraware/pdd-backend/pull/129 - [8e82194](https://github.com/terraware/terraware-server/pull/2135/commits/8e82194b0cf267bad6f049af83d1c8f35427a333)
https://github.com/terraware/pdd-backend/pull/130 - [1d0b5da](https://github.com/terraware/terraware-server/pull/2135/commits/1d0b5dafca80b4859c4878e00ef2e31cbd50de5b)
https://github.com/terraware/pdd-backend/pull/135

This pull request contains the result of each pull request from above, except [135](https://github.com/terraware/pdd-backend/pull/135). 135 will be included as a separate pull request, this is because it contains changes that have potentially broader impact than these changes.

Other changes:
- Test code was updated to remove the "canned" entity IDs and match our pre-existing patterns
- Where needed, "documentId" was swapped with "projectId", since values are now tied to projects

### CODE REVIEW

**Comment by nickgraz** [comment]
* **#2134** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/2134?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2135** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/2135?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nickgraz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/main/resources/db/migration/R__Comments.sql [hunk 1]
```diff
@@ -541,7 +541,6 @@ COMMENT ON TABLE accelerator.user_deliverable_categories IS 'Which deliverable c
 
 COMMENT ON TABLE accelerator.vote_options IS '(Enum) Available vote options.';
 
-
```
Comment (sgrimm) [inline src/main/resources/db/migration/R__Comments.sql]: The double newline here was to visually separate the sections for different schemas a little more. If we don't want that, we should remove the extra lines between other schemas too.

**Review by sgrimm — APPROVED** [review]
Ah, I totally failed to notice that the default values code was missing in the initial "merge the code bases" PR!

**Comment by nickgraz** [comment]
> Ah, I totally failed to notice that the default values code was missing in the initial "merge the code bases" PR!

Me neither, not a great mistake to make. I am lucky I wasn't further behind.

**Pull request merged**